kura/distrib/src/main/ant/build_equinox_distrib.xml

```diff
@@ -1349,6 +1367,34 @@ Resource-Processor: org.eclipse.kura.deployment.customizer.upgrade.rp.UpgradeScr
 		</zip>
 	</target>
 
+	<target name="tinyb-jars">
```

Comment:
I confess that I don't like this solution, but I didn't find a better one. However, if we can live with the libs for all the architectures in to the plugin folder, we can remove it.